orte/mca/rmaps/seq/rmaps_seq.c

```diff
@@ -189,6 +189,23 @@ static int orte_rmaps_seq_map(orte_job_t *jdata)
                     eptr--;
                 }
                 *(eptr+1) = 0;
+                /*
+                 * If the submitted LSF job has memory binding related resource requirement, after
+                 * the cpu id list there are memory binding options.
+                 *
+                 * The following is the format of LSB_AFFINITY_HOSTFILE file:
+                 *
+                 * Host1 0,1,2,3 0 2
+                 * Host1 4,5,6,7 1 2
+                 *
+                 * Each line includes: host_name, cpu_id_list, NUMA_node_id_list, and memory_policy.
+                 * In this fix we will drop the last two sections (NUMA_node_id_list and memory_policy)
+                 * of each line and keep them in 'membind_opt' for future use.
+                 */ 
+                if (NULL != (membind_opt = strchr(sep, ' '))) {
+                    *membind_opt = '\0';
+                    membind_opt++;
```

Comment:
Just curious - where is `membind_opt` actually used? I understand the termination of the `sep` string, but incrementing `membind_opt` implies that you intend to use it somewhere. Is there something missing, or perhaps the full patch isn't available yet?

It's fine as-is, just want to ensure it was intentional to leave this dangling.

Reply:
It's currently not used - and is documented as such. It may be used in the future if we add membind support.